evalml/automl/automl_search.py

```diff
@@ -763,10 +766,6 @@ def describe_pipeline(self, pipeline_id, return_dict=False):
         logger.info("Total training time (including CV): %.1f seconds" % pipeline_results["training_time"])
         log_subtitle(logger, "Cross Validation", underline="-")
 
-        if pipeline_results["high_variance_cv"]:
```

Comment:
I moved the logging behavior from `describe_pipeline` to `_add_result` but otherwise kept the same usage of `high_variance_cv` within rankings etc.. I feel like it is more appropriate to notify during the search process and not just when describing a pipeline. Happy to discuss further!

Reply:
I think this makes sense!

Reply:
Agreed!